README.md

```diff
@@ -1,42 +1,111 @@
-NOTE: This software is still in alpha phase and is not feature complete. Expect breaking changes and instability until version 2.1.0 is released
-
 # Datum
-Datum is a personal database for collecting and analyzing data on your life
 
-Record timestamped occur data for point-based time series data tracking:
-`datum occur alcohol type=beer`
-`datum occur vitamin -t 9:15am` # flexibly control time of occurence, relative and absolute
+> A command-line personal analytics database for life tracking and data analysis
+
+![Alpha Version](https://img.shields.io/badge/status-alpha-orange)
+
+Datum helps you collect, track, and analyze personal data through an intuitive command-line interface. It supports multiple types of time-series data collection and arbitrary data storage.
+
+## Features
+
+### Data Collection
+
+- **Point-in-time Events**
+  ```bash
+  datum occur alcohol type=beer
+  datum occur vitamin -t 9:15am  # Flexible timestamp control
+  ```
+
+- **Duration Tracking**
+  ```bash
+  datum start sleep
+  datum end sleep
+  ```
+
+- **State Management**
+  ```bash
+  datum switch project emails
+  datum switch project meeting
+  datum switch project end
+  ```
 
-Record start/end data for block-based time series data tracking:
-`datum start sleep`
-`datum end sleep`
+- **General Data Storage**
+  ```bash
+  datum add person name=Nate city=Berlin
+  ```
 
+### Data Visualization
 
-Record state based time series data:
-`datum switch project emails`
-`datum switch project meeting`
-`datum switch project end`
+Datum provides multiple ways to visualize and analyze your data:
 
+- **Daily Timeline View**
+  ```bash
+  datum dayview            # View today's tracked time blocks
+  datum dayview -d 2024-02-01  # View specific date
```

Comment:
`-d` doesn't actually exist as an option here yet, but it is a good idea, so I've added it to this PR